soft4pes/model/grid/rl_grid.py

```diff
@@ -0,0 +1,163 @@
+""" Model of a grid with stiff voltage source and RL-load in alpha-beta frame"""
+
+import numpy as np
+from soft4pes.util.conversions import abc_2_alpha_beta
+
+
+class RLGrid:
+    """
+    Model of a grid with stiff voltage source and RL-load in alpha-beta frame. 
+
+    Attributes
+    ----------
+    Vgr : float
+        Grid rated voltage [pu].
+    wg : float
+        Grid angular frequency [pu].
+    Rg : float
+        Resistance [pu].
+    Xg : float
+        Reactance [pu].
+    x : 1 x 2 ndarray of floats
+        Current state of the grid [pu]. The grid state is represented by the 
+        grid current in alpha-beta frame.
+    """
+
+    def __init__(self, Vgr, fgr, Rg, Lg, base):
```

Comment:
I do not fully agree. This would require an additional file, with just one function. Other systems would also require their own files (or one file could be used, but that would be messy). It would also not be user-friendly, as one additional function would have to be used 

The usage would then be:
```
base = baseGrid(values_in_SI)
pu_values = getGridPuValues(values_in_SI, base)
grid = RLGrid(pu_values)   
```

Reply:
each system should have a file for its parameters. In that file the p.u. derivation will be done. The model of the system should not have information of the parameters as it does not depend on this

Reply:
Ok, I will make a note about this. The arguments will be changed to p.u. when the parameter files are created. For now, lets have them in SI